docs/source/v1.9.md.inc

```diff
@@ -2,15 +2,22 @@
 
 ### :new: New features & enhancements
 
-- Added number of subject to `sub-average` report (#902, #910 by @SophieHerbst)
+- Added number of subjects to `sub-average` report (#902, #910 by @SophieHerbst)
 - The type annotations in the default configuration file are now easier to read: We
   replaced `Union[X, Y]` with `X | Y` and `Optional[X]` with `X | None`. (#908, #911 by @hoechenberger)
+- Added a new configuration option [`spatial_filter_raw`][mne_bids_pipeline._config.spatial_filter_raw]
+  to control whether to create cleaned raw data via ICA or SSP. Previously, we'd only clean epochs.
+  (#840, #926 by @larsoner and @hoechenberger)
 
 ### :warning: Behavior changes
 
 - All ICA HTML reports have been consolidated in the standard subject `*_report.html`
   file instead of producing separate files (#899 by @larsoner).
-- Changed default for `source_info_path_update` to `None`.  In `_04_make_forward.py`
+- When using ICA or SSP with resting-state data, we now automatically produce cleaned raw data files. This
+  behavior can be controlled via the new [`spatial_filter_raw`][mne_bids_pipeline._config.spatial_filter_raw]
+  configuration option. (#840, #926 by @larsoner and @hoechenberger)
```

Comment:
The introduction of writing raw data was actually in 1.6 / #840. So the behavior change here is actually to *stop* writing files by default. So maybe we should leave the default as `True` for backward compat. Then this PR is just to add the ability to have `False | None`.

Personally I think `True` is the most reasonable default anyway. If you ever want to re-epoch your data (I often do!) then you'll want those preprocessed raw files. And you can cross-check individual runs against the produced epochs, it's more complete to have raw data report at each stage, etc. even though it makes the report longer.